tests/WebAssembly/TestWithWebAssembly/Pages/Index.razor

```diff
@@ -0,0 +1,92 @@
+﻿@page "/"
+@using OmniSharp.WebAssembly
+@using System.Text.RegularExpressions
+@using System.Web
+
+<PageTitle>Index</PageTitle>
+
+<h1>Hello, world!</h1>
```

Comment:
just a simple page that allows you to upload a file and then prints out the console output